.github/workflows/rebuild-trivy-bundle.yaml

```diff
@@ -0,0 +1,261 @@
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: actions/checkout@v3
+        with:
+          repository: mesosphere/trivy-bundles
+          ref: main
+      
+      - name: Download Artifact with dkp-insights Release Branches and their trivy-bundles Image Tags
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: actions/download-artifact@v3
+        with:
+          name: trivy-tags-to-rebuild
+
+      - name: Create an Artifact for Saving Updated trivy-bundles Image Tags
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: mkdir -p artifacts && touch ./artifacts/updated-trivy-bundles-image-tags.txt
+      
+      - name: Login to Docker Hub
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_READ_WRITE_USERNAME }}
+          password: ${{ secrets.DOCKER_READ_WRITE_PASSWORD }}
+
+      - name: Rebuild and Push Updated trivy-bundles Images
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: |
+          while read branch_and_tag; do
+            dkp_insights_branch=${branch_and_tag%=*}
+            trivy_bundles_image_tag=${branch_and_tag##*=}
+            trivy_image_version=${trivy_bundles_image_tag%-*}
+            updated_time_stamp=$(date -u +%Y%m%dT%H%M%SZ)
+
+            echo "dkp_insights_branch: $dkp_insights_branch"
+            echo "trivy_bundles_image_tag: $trivy_bundles_image_tag"
+            echo "trivy_image_version: $trivy_image_version"
+            echo "time_stamp: $updated_time_stamp"
+
+            TRIVY_IMAGE_TAG=$trivy_image_version \
+              TIMESTAMP=$updated_time_stamp \
+              OUTPUT_IMAGE=$TRIVY_IMAGE_NAME \
+              make publish-trivy-bundled-image
+            
+            echo "${dkp_insights_branch}=${trivy_image_version}-${updated_time_stamp}" >> ./artifacts/updated-trivy-bundles-image-tags.txt
+
+          done <tags-to-rebuild.txt
+
+      - name: Upload Artifact with dkp-insights Release Branch Names and their Corresponding Updated trivy-bundles Image Tags
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: actions/upload-artifact@v3
+        with:
+          name: updated-trivy-bundles-image-tags
+          path: ./artifacts/updated-trivy-bundles-image-tags.txt
+
+
+  update-dkp-insights-release-branches:
+    needs: [rebuild-trivy-bundles-images, determine-if-workflow-should-run]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Reinstall Go for Current User
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: |
+          wget "https://go.dev/dl/$(curl 'https://go.dev/VERSION?m=text').linux-amd64.tar.gz"
+          sudo tar -C /usr/local -xzf go*.linux-amd64.tar.gz
```

Comment:
We don't actually end up building `dkp-insights` here (we did previously). 

Would it be possible to remove steps the build steps, involving use of  `working-directory: //usr/local/go/src/github.com/mesosphere/dkp-insights` to simplify this PR?

This workflow doesn't run too frequently or in a critical path so runtime isn't as much of a concern, but removing the need to be in `GOPATH` etc... should simplify this PR.